ACCS-365: Payment Services: Stored Cards at Checkout (#821)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins-b2b/requisition-list/functions.mdx

|---|---|---|---|
| `currentPage` | `number` | No | The page number for pagination (1-indexed). Used to navigate through multiple pages of requisition lists. |
| `pageSize` | `number` | No | The number of requisition lists to return per page. Controls how many lists appear on each page. |

> | `listItemsPageSize` | `number` | No | Sets how many items are loaded per list in each GraphQL request. The default is `100`. If a list has more than 100 items, additional requests are automatically made so users (like on a PDP “already on list” view) can see all items. |

Reviewer: ```suggestion
| `listItemsPageSize` | `number` | No | Sets the number of items loaded per GraphQL request (default: 100). If a list exceeds this, additional requests are made automatically so all items are available to the user. |
```
